app/controllers/api/v1/service_plans_controller.rb

```diff
@@ -14,8 +14,8 @@ def index
       end
 
       def create
-        svc = Catalog::ImportServicePlans.new(params.require(:portfolio_item_id))
-        render :json => svc.process.service_plans
+        Catalog::ImportServicePlans.new(params.require(:portfolio_item_id)).process
+        render :json => Catalog::ServicePlanJson.new(:portfolio_item_id => params[:portfolio_item_id], :collection => true).process.json
```

Comment:
Should ImportServicePlan just return this from the `process` method? Looks like we're not using `ImportServicePlan` any where else.

Reply:
Sure, I like that idea. Updated.